src/components/Accordion/index.tsx

```diff
@@ -121,3 +122,32 @@ export default function Accordion({
     </div>
   );
 }
+
+interface AccordionGroupProps {
+  list: Array<{ label: string; title: string; description: React.ReactNode }>;
+  size: AccordionProps["size"];
+}
+
+export function AccordionGroup({ list, size = "tablet" }: AccordionGroupProps) {
+  const [activeItemIndex, setActiveItemIndex] = useState<number | null>(null);
+
+  const handleItemClick = (key: number) => {
+    setActiveItemIndex((prev) => (prev === key ? null : key));
+  };
```

Comment:
```suggestion
  const handleItemClick = (key: number) => () => {
    setActiveItemIndex((prev) => (prev === key ? null : key));
  };
  
  onClick={handleItemClick(index)}
```

이런식으로 넣을 수도 있을 것 같습니다~

인라인으로 넣는걸 싫어하는 사람들도 많은 것 같아요

Reply:
아 그렇군요. 이 둘의 차이가 정확히 어떻게 될까요...? 싫어하는 사람이 많다면 그런 이유가 좀 궁금합니다

Reply:
onClick에 인라인으로 함수가 있으면 함수 확인하러 아래를 다 봐야하는 번거로움이 있어서 이렇게 활용하기도 합니다.

코드 통일성도 있긴하구요.

함수를 리턴하는 함수를 만들었다 보시면 됩니다.

Reply:
애초에 이벤트 핸들러에 인자를 넘겨주려 호출하는 목적이라, 인라인이라고는 해도 더 길어질 상황은 없을 거 같기는 해요. 제안하신 코드를 보면 결국 고차함수를 따로 만들어야 하는데 오히려 헷갈릴 수도 있겠다는 생각이 듭니다. 코드 통일성 측면에서는 저희 둘끼리 맞춰가면 괜찮을 거 같아요😄 일단 아까 이야기 나눴던 대로 현상 유지하고, 함 지켜보시죠!